master/internal/db/postgres_rp_workspace_bindings.go

```diff
@@ -39,3 +39,14 @@ func (db *PgDB) RemoveRPWorkspaceBindings(ctx context.Context,
 
 	return err
 }
+
+func (db *PgDB) OverwriteRPWorkspaceBindings(ctx context.Context, workspaceIds []int32, poolName string) error {
+	// Remove existing ones with this pool name
+	_, err := Bun().NewDelete().Table("rp_workspace_bindings").Where("pool_name = ?", poolName).Exec(ctx)
+	if err != nil {
+		return err
+	}
+
+	err = db.AddRPWorkspaceBindings(ctx, workspaceIds, poolName)
```

Comment:
Check err again?

Reply:
I think it should be fine because we'll return the error anyways